### PR TITLE
migrations script for materials table

### LINKED ISSUE #62: Knex Migrations for Material related tables
Create the Knex Migrations in order to create the DB schema for the following tables:

Table area_material
Table area_material_logs
Table material

### PR DESCRIPTION
**Pull request by mahieakhtar**
migrations script for material table
closes #62

### CODE REVIEW

**Comment by zkwsk** [comment]
Looks like this one is depending on #44

File: src/server/migrations/20200319154558_create_material_table.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('material', (table) => {
+    table.increments('id').primary();
+    table.string('name', 255).notNullable();
+    table.string('company_name', 255).notNullable();
```
Comment (mahieakhtar) [inline src/server/migrations/20200319154558_create_material_table.js]: Please delete the company_name

**Comment by mahieakhtar** [comment]
Maybe I have taken the latest from develop branch so as to sync my db-migration branch with develop branch and thus might have been added. There was also lots of effort to fix the eslint failing on pull request which was not failing at my local workspace.

File: src/server/migrations/20200327024126_create_materials_table.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('materials', function(table) {
+    table
+      .increments('id')
+      .primary()
```
Comment (mahieakhtar) [inline src/server/migrations/20200327024126_create_materials_table.js:5]: Relation is not there

**Comment by dpfernandes** [comment]
Approved

**Pull request merged**